modules/Data.py

```diff
@@ -106,3 +106,12 @@ def get_lending_currencies():
     for cur in lending_balances:
         currencies.append(cur)
     return currencies
+
+
+def truncate(f, n):
```

Comment:
No, keep the function as is, just add the link in a comment so we know where it came from an what the logic is. Not a simple one to understand by just reading it.